src/components/track.tsx

```diff
@@ -3,49 +3,84 @@ import * as React from "react";
 import { accentColor } from "../constants";
 import { usePublicFileURL } from "../hooks/use-public-file-url";
 
+const TitleContainer = styled.div`
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4.5rem;
+  overflow: hidden;
+  transition: 0.5s ease;
+
+  background-color: rgba(0, 0, 0, 0.75);
+
+  -webkit-backdrop-filter: blur(10px);
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+
+  z-index: 0;
+  cursor: default;
```

Comment:
from a UX point of view, since we're "interactive" now, is `cursor: default` still appropriate? I'm not sure, but `cursor: pointer` seems weird, too, since we can't click on the track

Reply:
After going through all of the valid options that the "cursor" option allows for, I think "default" actually makes the most sense.